enterprise/backend/src/metabase_enterprise/workspaces/isolation.clj

```diff
@@ -102,6 +105,7 @@
                              (let [isolated-table (duplicate-output-table! database workspace hydrated-output)]
                                (t2/insert! :model/WorkspaceMappingTable
                                            {:upstream_id   (:id upstream-output)
+                                            ;; Following can be nil in case table was not created
```

Comment:
I don't think this fk is nullable, so perhaps the best thing to do here is to skip the record.

We're not using these mappings in anger for tables (yet), so you can just leave a TODO for us to figure out how this gets backfilled once the table exists.

Reply:
Are you going to fix this?